File: src/js/beans/Plato.js
class Plato {

    // Propiedades
    id;
    nombre;
    descripcion;
    precio;
    disponible;
  
    // Metodos
    fromJson(json){
      this.id = json.id;
      this.nombre = json.nombre;
      this.descripcion = json.descripcion;
      this.precio = json.precio;
      this.disponible = json.disponible;
    }
  
    print(){
      console.log("Plato: { id: " + this.id +  ", nombre: " + this.nombre +  ", descripcion: " + this.descripcion + ", precio: " +  this.precio + ", disponible: " + this.disponible +" }");
    }
}

exports.Plato = Plato;

File: backend-arso/restaurantes/src/main/java/modelo/Plato.java
package modelo;

public class Plato {

	private String id;
	private String nombre;
	private String descripcion;
	private double precio;
	private String disponible;

	
	
	
	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getNombre() {
		return nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}

	public String getDescripcion() {
		return descripcion;
	}

	public void setDescripcion(String descripcion) {
		this.descripcion = descripcion;
	}

	public double getPrecio() {
		return precio;
	}

	public void setPrecio(double precio) {
		this.precio = precio;
	}
	
	public String getDisponible() {
		return disponible;
	}

	public void setDisponible(String disponible) {
		this.disponible = disponible;
	}

	@Override
	public String toString() {
		return "Plato [id=" + id + ", nombre=" + nombre + ", descripcion=" + descripcion + ", precio=" + precio
				+ ", disponible=" + disponible + "]";
	}
	
	



}

